Fastlane/deployment_lanes.rb

```diff
@@ -266,6 +266,25 @@
   release(hotfix: true)
 end
 
+desc 'Create a build for running tests on browser stack'
+lane :appium_build do |options|
+  scheme = options[:scheme] || ENV['XCODE_SCHEME']
+  clear_derived_data
+  # Set timeout to prevent xcodebuild -list -project to take to much retries.
+  ENV['FASTLANE_XCODEBUILD_SETTINGS_TIMEOUT'] = '120'
+  ENV['FASTLANE_XCODE_LIST_TIMEOUT'] = '120'
+
+  gym(
+    scheme: scheme,
+    configuration: 'Debug',
```

Comment:
Why does it need to be `Debug`?

Reply:
We are sending to browserstack where they resign the app to run in the cloud devices and want the tests to be run against the our staging environment. Wasn't sure if these can be done in Release configuration

Reply:
Right, yes for staging it needs to be debug